src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const styleStringToArr = sourceString.trim().split(';');
```

Comment:
There's no need to create a copy of the string before splitting it. The `split` method does not mutate the original string.